app/services/item_change_set_persister.rb

```diff
@@ -19,15 +19,22 @@ def update
     updated = update_structural(updated) if collection_ids_changed?
     updated = update_identification(updated) if source_id_changed? || catkey_changed?
     updated = updated_administrative(updated) if admin_policy_id_changed?
-    Dor::Services::Client.object(model.externalIdentifier).update(params: updated)
+    updated = updated_access(updated) if access_changed?
+    object_client.update(params: updated)
   end
 
   private
 
   attr_reader :model, :change_set
 
   delegate :collection_ids_changed?, :source_id_changed?, :catkey_changed?, :admin_policy_id_changed?,
-           :collection_ids, :source_id, :catkey, :admin_policy_id, to: :change_set
+           :collection_ids, :source_id, :catkey, :admin_policy_id, :license, :license_changed?,
+           :copyright_statement, :copyright_statement_changed?, :use_statement, :use_statement_changed?,
+           to: :change_set
+
+  def object_client
```

Comment:
based on our discussion a few weeks ago about naming, would this be better named cocina_object_client ?  (or was it cocina_item_client?)

Reply:
@ndushay so in this case, I think `object_client` is the least surprising name. see: https://github.com/search?q=org%3Asul-dlss+%22object_client%22&type=code

I think our naming changes were focused on cocina-models instances.